data-analysis/generation-data-analysis.py

```diff
@@ -0,0 +1,103 @@
+import math
```

Comment:
Your pull request: DATAANALYSIS-1021 Added dataanalysis
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']

Your commit: DATAANALYSIS-1021 Added main.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: DATAANALYSIS-1021 Added test.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: DATAANALYSIS-1021 Added generation-data-analysis.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']